### DISCUSSION TITLE
Nvidia 4090 2nd fan header issues

### DISCUSSION THREAD

**Discussion by SpaceballsTheGithubName** [Rem0o/FanControl.Releases#2312] [Q&A] [answered]
See the attched video and screenshot of my current settings. For whatever reason, I cannot get the second header (Gigabyte card, 3 fans, header 1 is fan closest to IO, header 2 is middle fan and third fan) on my GPU to match my curves, it constantly ramps up. I will be working when all of a sudden, the fans on the GPU ramp up instantly for a second or 2 then go  back down, not to mention the RPM is all over the place. Anyone have any ideas to get header 2 to match header 1? Header 1 has absolutely no issues.

![GPU graph](https://github.com/Rem0o/FanControl.Releases/assets/109682543/1fb5fed4-ffb8-4b9d-90b4-fb78486d0b25)

https://github.com/Rem0o/FanControl.Releases/assets/109682543/8270454d-7fc5-471b-bf87-771f10d96f75

**Comment by Rem0o** [comment]
My guess is your start/stop% is wrong and you are trying to run the card at too low of a RPM.   You are running your card here at 45%, which is at the edge of stopping.  In this case I think 42 is set to low, as we can see at 45 it still dwindle downs to 0, before FanControl kicks it back up at 48 ( start ) to try and hold it back at 45, where it will slow down again. 

Basically fans need a minimum amount of power to hold rotation. Here we apply 45% power, but that just doesn't fulfill the requirement to fight inertia and its mass to hold a steady RPM.

**Comment by SpaceballsTheGithubName** [comment]
Thanks for the quick response. 

I would have agreed with you if not for the first fan header to be maintaining its RPM and speed consistently, this is ONLY affecting Fan header 2, and I did run the initial startup procedure with FanControl after wiping my old UserConfig. This is a relatively new problem as it was not happening about 3 to 4 weeks ago. I thought it was maybe the 0RPM mode being off in the settings but that did not fix the issue either. 

To be frank I also dont think this is FanControl, I believe it is an issue with Nvidia and their 0RPM mode, but i'm mainly looking to see if anyone else has had this issue or maybe has experienced this. 

Also i did try and adjust my start% and stop%, i threw them up to 60 and the behavior did not change.

**Reply by Rem0o** [reply]
Both fans can be a little different and again, you are running them at the edge in your video. If instead of 45 your fan curve has a minimum % of like 50, do you get the same effect?

**Reply by SpaceballsTheGithubName** [reply, marked as answer]
Looks like you were right. with your update today and setting the curve to 50% i no longer get the effect, see the video. Odd that the fans have wildly different RPMs when both set to the same curve. Thanks for the information, didnt expect the fans to be so much different/

https://github.com/Rem0o/FanControl.Releases/assets/109682543/e63e0663-232a-46c9-9b04-1fe7fdd7c152
